cypress/e2e/tests/migration/reports-tab/sort.test.ts

```diff
@@ -0,0 +1,122 @@
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import {
+    clickOnSortButton,
+    createMultipleApplications,
+    createMultipleStakeholders,
+    deleteAllMigrationWaves,
+    deleteApplicationTableRows,
+    getTableColumnData,
+    login,
+    verifySortAsc,
+    verifySortDesc,
+} from "../../../../utils/utils";
+import { legacyPathfinder, SEC, SortType } from "../../../types/constants";
+import { Stakeholders } from "../../../models/migration/controls/stakeholders";
+import { Application } from "../../../models/migration/applicationinventory/application";
+import { AssessmentQuestionnaire } from "../../../models/administration/assessment_questionnaire/assessment_questionnaire";
+import { Reports } from "../../../models/migration/reports-tab/reports-tab";
+import {
+    IdentifiedRiskTableHeaders,
+    questionnaireNameColumnDataLabel,
+} from "../../../views/reportsTab.view";
+
+let stakeholder: Stakeholders;
+let application: Application;
+
+const sortableColumns = [
+    IdentifiedRiskTableHeaders.questionnaireName,
+    IdentifiedRiskTableHeaders.section,
+    IdentifiedRiskTableHeaders.answer,
+];
+
+// Automates Polarion TCs 452
+describe(["@tier2"], "Reports tests", () => {
+    before("Login and Create Test Data", function () {
+        login();
+        deleteAllMigrationWaves();
+        deleteApplicationTableRows();
+        AssessmentQuestionnaire.deleteAllQuesionnaire();
+        AssessmentQuestionnaire.enable(legacyPathfinder);
+        stakeholder = createMultipleStakeholders(1)[0];
+        application = createMultipleApplications(1)[0];
+        application.perform_assessment("high", [stakeholder]);
+        application.verifyStatus("assessment", "Completed");
+        application.perform_review("high");
+        application.verifyStatus("review", "Completed");
+        Reports.open(100);
+    });
+
+    sortableColumns.forEach((column: string) => {
+        it(`${column} sort validations`, function () {
+            let columnDataLabel = column;
+            if (column === (IdentifiedRiskTableHeaders.questionnaireName as string)) {
+                columnDataLabel = questionnaireNameColumnDataLabel;
+            }
+
+            const unsorted = getTableColumnData(columnDataLabel);
+
+            clickOnSortButton(column, SortType.ascending);
+            verifySortAsc(getTableColumnData(columnDataLabel), unsorted);
+
+            clickOnSortButton(column, SortType.descending);
+            verifySortDesc(getTableColumnData(columnDataLabel), unsorted);
+        });
+    });
+
+    it("Risk sort validation", function () {
```

Comment:
May be you can re-use my function for sorting? Please consider it, my PR is merged already

Reply:
updated